presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -590,10 +592,32 @@ private void testCreateTableAs(Session session, HiveStorageFormat storageFormat)
     @Test
     public void testCreatePartitionedTableAs()
     {
-        testWithAllStorageFormats(this::testCreatePartitionedTableAs);
+        testWithAllStorageFormats(this::createPartitionedTableAs);
     }
 
-    private void testCreatePartitionedTableAs(Session session, HiveStorageFormat storageFormat)
+    @Test
+    public void testPropertiesTable()
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_show_properties" +
+                " WITH (" +
+                "format = 'orc', " +
+                "partitioned_by = ARRAY[ 'SHIP_PRIORITY', 'ORDER_STATUS' ]," +
+                "orc_bloom_filter_columns = ARRAY[ 'SHIP_PRIORITY', 'ORDER_STATUS' ]," +
+                "orc_bloom_filter_fpp = 0.5" +
+                ") " +
+                "AS " +
+                "SELECT orderkey AS order_key, shippriority AS ship_priority, orderstatus AS order_status " +
+                "FROM tpch.tiny.orders";
+
+        assertUpdate(createTable, "SELECT count(*) FROM orders");
+        BasicQueryInfo createTableQuery = ((DistributedQueryRunner) getQueryRunner()).getCoordinator().getQueryManager().getQueries().stream().filter(basicQueryInfo -> basicQueryInfo.getQuery().equals(createTable)).findFirst().get();
```

Comment:
This is a clever way to get the query ID and node version! However, there is a simpler way:
```java
String queryId = (String) computeScalar("SELECT query_id FROM system.runtime.queries WHERE query LIKE 'CREATE TABLE test_show_properties%'");
String nodeVersion = (String) computeScalar("SELECT node_version FROM system.runtime.nodes WHERE coordinator");